resolwe/storage/manager.py

```diff
@@ -79,7 +102,7 @@ def copy_single_location(self, file_storage: FileStorage, connector_name: str):
             storage_location.transfer_data(new_storage_location)
             new_storage_location.status = StorageLocation.STATUS_DONE
             new_storage_location.save()
-        except Exception:
```

Comment:
Why have you removed this?

Reply:
I believe it is clearer to just write `except` instead of `except Exception`. I know they are not 100% compatible but that should make no difference in this situation.

Reply:
Actually, it does matter. Especially as this is a long-running task, and the except (without `Exception`) also catches `SystemExit` and `KeyboardInterrupt`, so the program cannot be interupted.